tests/functional/test_notebooks.py

```diff
@@ -176,6 +176,8 @@
 #                                         params=None)
 #         assert result.exit_code == 0
 #
+# TODO: Add test case for creating notebook with tags
+#
```

Comment:
Looks like You forgot about something ;)

Reply:
I passed it because there are no create functionality (it is but it is commented) and this part of test are also commented for some reason. That is why I wanted to avoid adding commented test without possibility to test functionality